hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -131,6 +131,9 @@ public static void createHoodieProperties(FileSystem fs, Path metadataFolder, Pr
         // Use latest Version as default unless forced by client
         properties.setProperty(HOODIE_TIMELINE_LAYOUT_VERSION, TimelineLayoutVersion.CURR_VERSION.toString());
       }
+      if (!properties.containsKey(HOODIE_BASE_FILE_FORMAT_PROP_NAME)) {
+        properties.setProperty(HOODIE_BASE_FILE_FORMAT_PROP_NAME, DEFAULT_BASE_FILE_FORMAT.name());
+      }
```

Comment:
Can you also add a getBaseFileFormat() API which returns PARQUET if not set in hoodie.properties.

Reply:
hi @bvaradar, it already exists in master branch.
```
// HoodieTableConfig#getBaseFileFormat

/**
 * Get the base file storage format.
 *
 * @return HoodieFileFormat for the base file Storage format
 */
public HoodieFileFormat getBaseFileFormat() {
  if (props.containsKey(HOODIE_BASE_FILE_FORMAT_PROP_NAME)) {
    return HoodieFileFormat.valueOf(props.getProperty(HOODIE_BASE_FILE_FORMAT_PROP_NAME));
  }
  if (props.containsKey(HOODIE_RO_FILE_FORMAT_PROP_NAME)) {
    return HoodieFileFormat.valueOf(props.getProperty(HOODIE_RO_FILE_FORMAT_PROP_NAME));
  }
  return DEFAULT_BASE_FILE_FORMAT;
}
```